MSAL/test/app/mac/MSALAcquireTokenViewController.m

```diff
@@ -467,4 +524,41 @@ - (MSALAccount *)selectedAccount
     return self.accounts[self.userPopup.indexOfSelectedItem-1];
 }
 
+- (MSALInteractiveTokenParameters *)tokenParamsWithSSOSeeding:(BOOL)isSSOSeedingCall
+{
+    MSALTestAppSettings *settings = [MSALTestAppSettings settings];
+    MSALInteractiveTokenParameters *parameters = [[MSALInteractiveTokenParameters alloc] initWithScopes:isSSOSeedingCall ? [MSALTestAppSettings getSSOSeedingScope] : [settings.scopes allObjects]
+                                                                                      webviewParameters:[self msalTestWebViewParameters]];
+    
+    if (isSSOSeedingCall)
+    {
+        parameters.authenticationScheme = [MSALAuthenticationSchemeBearer new];
+    }
+    else
+    {
+        parameters.authenticationScheme = [self authScheme];
+    }
+    
+    parameters.loginHint = [self.loginHintTextField stringValue].length ? [self.loginHintTextField stringValue] : nil;
+    parameters.account = settings.currentAccount;
+    parameters.promptType = isSSOSeedingCall ? MSALPromptTypeDefault : [self promptType];
```

Comment:
any reason why we shouldn't let promptType be controller even for SSO seeding call?

Reply:
I didn't test all the prompt types to see the difference, but in seeding params we already have "prompt=none" so I guess it should replace whatever the promptType value here.